MaintenanceDashboard.Common/Text.resx

```diff
@@ -0,0 +1,132 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
To jest plik resx o którym wspominałem. w Visual Studio generują się one automatycznie w kreatorze przy `Add File`, wystarczy, że w szukajce wpiszesz `res`